examples/reference/widgets/Debugger.ipynb

```diff
@@ -3,33 +3,32 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "b15a6cf0",
```

Comment:
@Hoxbro do we allow `id`s now in cells metadata? Just to know.

Reply:
We don't really use the shell script anymore, I rely exclusively on Simon's excellent pre-commit hook. But I also thought that removed ids.

Reply:
I still have to use it on `examples` as there are notebooks that need to be pushed with their cell output, unfortunately :)